.travis.yml

```diff
@@ -32,20 +32,31 @@ install:
   - pip install -r rootfs/dev_requirements.txt
   - make -C docs/ deps
   - GLIDE_HOME=/home/travis/.glide make -C client bootstrap
+  - make prep-bintray-json
```

Comment:
we could, but then it'd get called twice; once for each deploy target. I figured we only needed to call it once.